shell/platform/android/android_shell_holder.cc

```diff
@@ -135,7 +139,27 @@ AndroidShellHolder::AndroidShellHolder(
         FML_LOG(ERROR) << "Failed to set Workers task runner priority";
       }
     });
+
+#if defined(__ANDROID_API__) && __ANDROID_API__ >= 28
```

Comment:
This will never be true.

We set the __ANDROID_API__ to 16 or 24 depending on the ABI we support. I think we want to just remove this.

Reply:
Removed this.